duckduckgo/iOS
# Add "Add to Homescreen" option

Task/Issue URL: https://app.asana.com/0/414709148257752/1204277537742323/f

**Description**:
Add the option to save links into the iOS homescreen.

**Steps to test this PR**:
1. Check if you're on iOS 16.4 +
2. Tap on the share sheet.
3. Find the "Add to homescreen" button and tap it
4. Verify if link was correctly saved into your homescreen
5. Check if tapping the icon opens DDG in the correct page when DDG is your default browser
6. Check if tapping the icon opens Safari in the correct page when Safari is your default browser

1. On iOS older than 16.4
2. Check that this new option does not appear

**Device Testing**:

* [ ] iPhone 14 

**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
